### PR TITLE
Fix outdated comment in test wrapper generator

### PR DESCRIPTION
**Pull request by domenic**

### CODE REVIEW

**Comment by w3c-bots** [comment]
[View the complete job log.](https://travis-ci.org/w3c/web-platform-tests/jobs/258767251)

# Firefox (nightly) #

**Comment by w3c-bots** [comment]
[View the complete job log.](https://travis-ci.org/w3c/web-platform-tests/jobs/258767252)

# Sauce (safari) #

**Comment by w3c-bots** [comment]
[View the complete job log.](https://travis-ci.org/w3c/web-platform-tests/jobs/258767258)

# Chrome (unstable) #

**Comment by w3c-bots** [comment]
[View the complete job log.](https://travis-ci.org/w3c/web-platform-tests/jobs/258767259)

# Sauce (MicrosoftEdge) #

File: streams/generate-test-wrappers.js [hunk 1]
```diff
@@ -1,18 +1,15 @@
...
 //
 // It will turn any importScripts inside the .js file into <script>s in the browser context wrapper.
 //
-// This could become obsolete if all of the following happen:
-// - https://github.com/w3c/web-platform-tests/issues/4210 gets fixed, allowing .any.js to work with all four contexts
-// - We find some way to include scripts (<script>/importScripts) in .any.js files
-// - Chrome becomes able to run .any.js tests on its infrastructure
-//   (https://bugs.chromium.org/p/chromium/issues/detail?id=653514)
+// This could become obsolete if https://github.com/w3c/web-platform-tests/issues/4210 gets fixed, allowing
+// .any.js to work with all four contexts
```
Comment (tyoshino) [inline streams/generate-test-wrappers.js]: let's add a period at the end using this opportunity

File: streams/generate-test-wrappers.js [hunk 2]
```diff
@@ -1,18 +1,15 @@
...
 // - js-filename1.dedicatedworker.html
-// - js-filename1.serviceworker.html
+// - js-filename1.serviceworker.https.html
 // (for each passed filename)
 //
 // It will turn any importScripts inside the .js file into <script>s in the browser context wrapper.
 //
-// This could become obsolete if all of the following happen:
-// - https://github.com/w3c/web-platform-tests/issues/4210 gets fixed, allowing .any.js to work with all four contexts
-// - We find some way to include scripts (<script>/importScripts) in .any.js files
```
Comment (tyoshino) [inline streams/generate-test-wrappers.js]: What's the reason we can remove this too now?

**Review by tyoshino — APPROVED** [review]

File: streams/generate-test-wrappers.js (hunk 2 above)
Comment (domenic) [inline streams/generate-test-wrappers.js]: The // META comments at http://web-platform-tests.org/writing-tests/testharness.html; I guess I was not aware of them when writing this.

Comment (tyoshino) [inline streams/generate-test-wrappers.js]: Oh, got it! lgtm

**Pull request merged**